### PR TITLE
Clarify and complete installation process

### PR DESCRIPTION
**Pull request by j33f**

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -129,67 +145,88 @@ From the root directory:
...
-    $ npm install kuzzle
-    ```
+* A service [Elasticsearch](https://www.elastic.co/products/elasticsearch) running on localhost:9200
+* A service [Redis](http://redis.io/) running on localhost:6379
+* A properly installed [nodeJs](https://nodejs.org/en/download/package-manager/) **version 4** or upper
+
+### Step one
 
-* GIT
+Retrieve the Kuzzle source code from the [GitHub repo](https://github.com/kuzzleio/kuzzle.git) by doing
```
Comment (scottinet) [inline docs/installation.md]: Remove `by doing`

File: docs/installation.md [hunk 2]
```diff
@@ -129,67 +145,88 @@ From the root directory:
...
 
-    ```
     $ git clone https://github.com/kuzzleio/kuzzle.git
     $ cd kuzzle
-    ```
 
-## Manual install
+then install the dependencies by doing:
```
Comment (scottinet) [inline docs/installation.md]: Same here, simply add a semicolon instead of `by doing`

File: docs/installation.md [hunk 3]
```diff
@@ -129,67 +145,88 @@ From the root directory:
...
 
-### Default
+    $ npm install
 
-Prerequisites:
+### Step two
 
-* A service [Elasticsearch](https://www.elastic.co/products/elasticsearch) running on localhost:9200
-* A service [Redis](http://redis.io/) running on localhost:6379
+Configure your environment. Kuzzle has been designed to be launched into a container, so the default hosts used to access to the ElasticSearch and Redis servers needs to be tweaked to hit the right hosts. If everything is hosted on localhost, you can use environment variable to overwrite default ones:
```
Comment (scottinet) [inline docs/installation.md]: `...to be launched from inside a container...`

File: docs/installation.md [hunk 4]
```diff
@@ -129,67 +145,88 @@ From the root directory:
...
 
-To get a list of available options, you can run:
 
-```bash
-$ kuzzle start -h
-```
+### Step three
 
-You may also start Kuzzle by using [PM2](https://github.com/Unitech/pm2):
+Install the default plugins, by doing:
```
Comment (scottinet) [inline docs/installation.md]: Remove `by doing`

File: docs/installation.md [hunk 5]
```diff
@@ -129,67 +145,88 @@ From the root directory:
...
-```bash
-$ pm2 start app-start.js
-```
+    $ ./bin/kuzzle.js install
 
 
-### Change external services hosts
+### Finally
+
+Start Kuzzle with default options (one server and one worker) by doing
```
Comment (scottinet) [inline docs/installation.md]: Remove `by doing`

The default `kuzzle start` runs 1 thread executing both a server and a worker. This is not necessarily the best way to run Kuzzle, except on a machine with very limited resources.

I would have written: 
```
Start a server instance:
$ bin/kuzzle.js start --server

And then as many worker instances as you want. At least 1 worker is required:
$ bin/kuzzle.js start --worker

For more information, you can execute:
$ bin/kuzzle.js start --help
```

**Comment by ballinette** [comment]
+1

File: docs/installation.md [hunk 6]
```diff
@@ -129,67 +145,97 @@ From the root directory:
...
+
+
+### Finally
 
+Start a server instance:
 
-### Change external services hosts
+    $ ./bin/kuzzle.js start --server
+
+And then as many worker instances as you want. At least 1 worker is required:
```
Comment (xbill82) [inline docs/installation.md]: `...then launch as many...`

File: docs/installation.md [hunk 7]
```diff
@@ -129,67 +145,97 @@ From the root directory:
...
+
+    $ bin/kuzzle.js start --worker
+
+For more information, you can execute:
+
+    $ bin/kuzzle.js start --help
+
+### All steps in one
```
Comment (xbill82) [inline docs/installation.md]: `Altogether` ?

**Comment by xbill82** [comment]
:+1:

**Pull request merged**